src/states/infinitepeps.jl

```diff
@@ -157,10 +157,35 @@ end
 
 ## Vector interface
 
-function VectorInterface.scalartype(::Type{NT}) where {NT<:InfinitePEPS}
-    return scalartype(eltype(NT))
+VI.scalartype(::Type{NT}) where {NT<:InfinitePEPS} = scalartype(eltype(NT))
+VI.zerovector(A::InfinitePEPS) = InfinitePEPS(zerovector(unitcell(A)))
+
+function VI.scale(ψ::InfinitePEPS, α::Number)
+    _scale = Base.Fix2(scale, α)
+    return InfinitePEPS(map(_scale, unitcell(ψ)))
+end
+function VI.scale!(ψ::InfinitePEPS, α::Number)
+    _scale! = Base.Fix2(scale!, α)
+    foreach(_scale!, unitcell(ψ))
+    return ψ
+end
+function VI.scale!(ψ₁::InfinitePEPS, ψ₂::InfinitePEPS, α::Number)
+    _scale!(x, y) = scale!(x, y, α)
+    foreach(_scale!, unitcell(ψ₁), unitcell(ψ₂))
+    return ψ₁
+end
+VI.scale!!(ψ::InfinitePEPS, α::Number) = scale!(ψ, α)
```

Comment:
Do we want to add the out-of-place `scale!(y, x, alfa)` as well while we're at it?